cmd/stack.go

```diff
@@ -350,11 +351,28 @@ func printStatus(cmd *cobra.Command, servicesStatus []stack.ServiceStatus) {
 		return
 	}
 	t := table.NewWriter()
-	t.AppendHeader(table.Row{"Service", "Version", "Status"})
+	t.AppendHeader(table.Row{"Service", "Version", "Status", "Image Build Date", "VCS Ref"})
 
 	for _, service := range servicesStatus {
-		t.AppendRow(table.Row{service.Name, service.Version, service.Status})
+		t.AppendRow(table.Row{service.Name, service.Version, service.Status, formatTime(service.Labels.BuildDate), truncate(service.Labels.VCSRef, 10)})
```

Comment:
When creating a serverless project, there are no local containers at least for elasticsearch, kibana. For that scenario, `service.Labels` field is nil.


Here there is an example of the panic error that it is raised because of that:

```
 $ elastic-package stack status
2024/09/19 17:21:03  WARN Elastic Serverless provider is in technical preview
Status of Elastic stack services:
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x30 pc=0x3516bd3]

goroutine 1 [running]:
github.com/elastic/elastic-package/cmd.printStatus(0xc000546c08, {0xc0008fc460, 0x4, 0x0?})
	/home/mariorodriguez/Coding/work/elastic-package-stack-status-vcs-ref/cmd/stack.go:357 +0x233
github.com/elastic/elastic-package/cmd.setupStackCommand.func6(0xc000546c08, {0x5ba26a0?, 0x0?, 0x0?})
	/home/mariorodriguez/Coding/work/elastic-package-stack-status-vcs-ref/cmd/stack.go:299 +0x167
github.com/spf13/cobra.(*Command).execute(0xc000546c08, {0x5ba26a0, 0x0, 0x0})
	/home/mariorodriguez/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:985 +0xaca
github.com/spf13/cobra.(*Command).ExecuteC(0xc00054af08)
	/home/mariorodriguez/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:1117 +0x3ff
github.com/spf13/cobra.(*Command).Execute(0xc000078008?)
	/home/mariorodriguez/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:1041 +0x13
main.main()
	/home/mariorodriguez/Coding/work/elastic-package-stack-status-vcs-ref/main.go:26 +0x66
```

Reply:
This would be an example of that command with one of the latest elastic-package versions:
```
 $ elastic-package stack status 
2024/09/19 17:36:22  WARN Elastic Serverless provider is in technical preview
Status of Elastic stack services:
╭───────────────┬────────────────────────────┬───────────────────╮
│ SERVICE       │ VERSION                    │ STATUS            │
├───────────────┼────────────────────────────┼───────────────────┤
│ elasticsearch │ serverless (observability) │ healthy           │
│ kibana        │ serverless (observability) │ healthy           │
│ fleet         │ serverless (observability) │ healthy           │
│ elastic-agent │ 8.15.0                     │ running (healthy) │
╰───────────────┴────────────────────────────┴───────────────────╯
```